var/spack/repos/builtin/packages/py-thinc/package.py

```diff
@@ -10,7 +10,7 @@ class PyThinc(PythonPackage):
     homepage = "https://github.com/explosion/thinc"
     url      = "https://pypi.io/packages/source/t/thinc/thinc-7.4.1.tar.gz"
 
-    version('7.4.1', sha256='523e9be1bfaa3ed1d03d406ce451b6b4793a9719d5b83d2ea6b3398b96bc58b8')
+    version('7.4.1', sha256='0139fa84dc9b8d88af15e648fc4ae13d899b8b5e49cb26a8f4a0604ee9ad8a9e')
```

Comment:
Any idea where the previous checksum came from? As far as I know, it's not possible for PyPI checksums to change. Can you contact the developers?

Reply:
I'm guessing the original hash was a typo since the hash for 7.4.1 and 7.4.0 were originally the same? I'll poke @aweits